recipes/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.5" %}
+{% set sha256 = "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855" %}
```

Comment:
This file is wrong. It's a copy of https://github.com/bioconda/bioconda-recipes/pull/13872/files#diff-56ec1f76a52dd5b54362e8d356fa2ce4 but in the wrong folder. This is why you get this strange error.

Reply:
Thank you. I removed the mis-placed meta.yaml file from recipes/. I also modified the meta.yaml under recipes/multiphate/ and removed the build.sh file. More simple tests have been added to meta.yaml under test: commands:. I hope this is now correct and improved.